pkg/option/config.go

```diff
@@ -2346,6 +2362,9 @@ func (c *DaemonConfig) Populate() {
 
 	c.populateDevices()
 
+	c.IPAMClusterPoolV4CIDR = viper.GetStringSlice(IPAMClusterPoolV4CIDR)
+	c.IPAMClusterPoolV6CIDR = viper.GetStringSlice(IPAMClusterPoolV6CIDR)
```

Comment:
a small not here, it's probably worth it make a small validation the the cidrs passed. If `net.ParseCIDR` would not return an error that would be sufficient.

Reply:
ok, will address your comments._

Reply:
Done.